src/components/main/Scenario.js

```diff
@@ -10,80 +10,132 @@ class Scenario extends React.Component {
             results:"",
             staffNum:"The Results:",
             num:"",
+			center:{"text-align":'center'},
             staffs: [],
-			unit: null,
-			hppd: null,
-			numbeds: null,
-			census:100, // default to 100
+            info:{
+            	   unit:"",
+            	   HPPD:"",
+                bedUnit:"",
+                census:"",
+               }
         };
     }
 
-    //We are going to need to have alerts or errors so that we can test incorrect 
-    //input as of right now there is not a good way to test bad input in the scenario.test.js
-    // submitHandler = (event) => {
-    //     event.preventDefault();
-    //     let unit = this.state.unit;
-    //     let hppd = this.state.hppd;
-    //     let numbeds = this.state.numbeds;
-
-    //     if (!Number(hppd) && !Number(numbeds)) {
-    //         alert("Your hppd must be a number");
-    //         alert("Your numbeds must be a number");
-    //     }
-    //     if(Number(unit)){
-    //         alert("Your unit can't be a number")
-    //     }
-
-    // }
-
     changeHandler = (event) => {
         let name = event.target.name;
         let val = event.target.value;
-		if(name!=='unit') {
-			if(!(/^\+?[1-9][0-9]*$/.test(val)))
-			{
-				alert("Only numbers(positive integers) can be entered");
-				return;
-			}
-		}
-		this.setState({ [name]: val }, () => 
```

Comment:
This has to be in here to set the state accurately. Otherwise, it only sets 1 number at a time. For instance if you set the hppd to 12, it'll only grab the 1 at first and then will later grab the full number

Reply:
I studied it carefully, you should be the way to print the log, right? 
I don’t quite understand your intentions with this method